claudio-plugin/skills/gws-calendar-reader/tests/conftest.py

```
@@ -0,0 +1,96 @@
+        def __init__(self):
+            self.tmp_path = tmp_path
+            self.log_file = log_file
+            self.responses_file = responses_file
+
+        def set_responses(self, responses: dict) -> None:
+            self.responses_file.write_text(json.dumps(responses))
+
+        def read_log(self) -> list[list[str]]:
+            if not self.log_file.exists():
+                return []
+            return [json.loads(l) for l in self.log_file.read_text().splitlines() if l.strip()]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)